packages/@pollyjs/adapter-fetch/tests/integration/adapter-test.js

```diff
@@ -21,3 +21,36 @@ describe('Integration | Fetch Adapter', function() {
   adapterTests();
   adapterBrowserTests();
 });
+
+describe('Integration | Fetch Adapter | Concurrency', function() {
+  it('should prevent concurrent fetch adapter instances', function() {
+    const one = new Polly('one');
+    const two = new Polly('two');
+
+    one.connectTo(FetchAdapter);
+
+    expect(function() {
+      two.connectTo(FetchAdapter);
```

Comment:
`connectTo` vs providing the adapter via `configure` so that the constructor assignment happens and we can do cleanup/`stop()`.